### PR TITLE
Major update

### PR DESCRIPTION
**Pull request by troelslenda**
Repository is basically dead since site was migrated as a install profile on aegir. https://github.com/kdb/lydia

This is however now moved to different host and using make files to build sites, proved to be a bad decision, which is why everything is merged into one massive git repository OMG.

This Pull request also included a minor tracking script, as a library inserted in the lydias template.

### CODE REVIEW

**Comment by mikl** [comment]
Vildt 😃👍🏻

**Comment by googletorp** [comment]
Det her vækker minder :)

File: profiles/lydia/libraries/webtrends/scripts/webtrends.load.js [hunk 1]
```diff
@@ -23,6 +23,6 @@ window.webtrendsAsyncInit=function(){
         }).track();
 };
 (function(){
-    var s=document.createElement("script"); s.async=true; s.src="/scripts/webtrends.min.js";    
+    var s=document.createElement("script"); s.async=true; s.src="/profiles/lydia/libraries/webtrends/scripts/webtrends.min.js";
```
Comment (tripox) [inline profiles/lydia/libraries/webtrends/scripts/webtrends.load.js]: @troelslenda - the directory doesn't exist?

File: profiles/lydia/themes/lydias/template.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * Implements hook_preprocess_html().
+ */
+function lydias_preprocess_html(&$vars) {
+  drupal_add_js(libraries_get_path('webtrends'). '/scripts/webtrends.load.js');
```
Comment (tripox) [inline profiles/lydia/themes/lydias/template.php]: @troelslenda - where's this file?

**Pull request merged**